### PR TITLE
Cache the vectorization break point in schedulers

### PR DESCRIPTION
**Pull request by csarofeen**
Cache the vectorization break point as it's a compile time value.

### CODE REVIEW

**Comment by csarofeen** [comment]
!build

**Comment by csarofeen** [comment]
!build

**Comment by csarofeen** [comment]
!build

**Review by liqiangxl — APPROVED** [review]
LGTM.

File: csrc/scheduler/utils.cpp [hunk 1]
```diff
@@ -636,6 +639,11 @@ PersistentBufferInfo persistentBuffers(Fusion* fusion) {
     }
   }
 
+  // Projection analysis below
+  if (persistent_buffer_info.projectable_persistent_buffers.empty()) {
```
[the remark this answers is not in the document]
Comment (csarofeen) [reply csrc/scheduler/utils.cpp:643]: Is this a nit or important?

[the remark this answers is not in the document]
Comment (liqiangxl) [reply csrc/scheduler/utils.cpp:643]: Just a nit to save the following projection analysis when there are view ops.

**Pull request merged**